src/components/code_editor/code_editor.test.tsx

```diff
@@ -128,8 +128,13 @@ describe('OuiCodeEditor', () => {
           stopPropagation: () => {},
           key: keys.ESCAPE,
         });
-        const hint = findTestSubject(component, 'codeEditorHint').getDOMNode();
-        expect(hint).toBe(document.activeElement);
+        setTimeout(() => {
+          const hint = findTestSubject(
+            component,
+            'codeEditorHint'
+          ).getDOMNode();
+          expect(hint).toBe(document.activeElement);
```

Comment:
It looks like there are some inconsistencies with how we get the element in tests like this. Is there a reason for that? I.e. splitting it out into a separate variable, etc.

Reply:
No, I don't think there's any meaningful reason beyond different tests written by different people at different times. Mostly wanted this PR to have as minimal changes as possible to avoid breaking/changing the tests we've been relying on.